app/src/main/java/protect/card_locker/DBHelper.java

```diff
@@ -16,12 +16,15 @@
 import java.util.Arrays;
 import java.util.Currency;
 import java.util.Date;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
 
 public class DBHelper extends SQLiteOpenHelper {
     public static final String DATABASE_NAME = "Catima.db";
     public static final int ORIGINAL_DATABASE_VERSION = 1;
     public static final int DATABASE_VERSION = 16;
+    public static final int DEFAULT_ZOOM_LEVEL = 100;
```

Comment:
I understand why you want this variable but I'm not sure if this is the correct solution because with your changes:

1. Changing this will create a difference in database behaviour between old and new users
2. Changing this will change an old database migration (which is a really bad thing to do in general, if a migration is bad a new migration should be added to fix it instead)

I do get the new flow requires you to have this value somewhere because you need it while instantiating a `LoyaltyCard` so maybe just:

```
    public static final int DEFAULT_ZOOM_LEVEL = 100; // Do not change this variable without writing a new migration
```

and undo your change in the version 14 migration?

Reply:
You're right, I should not have touched the migration. Will fix.